src/app/components/RiskScoreCard.jsx

```diff
@@ -1,13 +1,49 @@
 import { Shield } from "lucide-react";
 import { Card } from "./ui/card";
 
-export function RiskScoreCard() {
+const severityColorMap = {
+  High: {
+    text: "text-red-600",
+    bg: "bg-red-100",
+    bar: "#D82000",
+  },
+  Medium: {
+    text: "text-orange-500",
+    bg: "bg-orange-100",
+    bar: "#FFA500",
+  },
+  Low: {
+    text: "text-green-600",
+    bg: "bg-green-100",
+    bar: "#32CD32",
+  },
+};
+
+export function RiskScoreCard({ riskScore = 0, severityLevel = "Low" }) {
+  const colors = severityColorMap[severityLevel] || severityColorMap.Low;
```

Comment:
use `??` instead of `||` for all cases where you expect the value to be `null` or `undefined`